README.md

```diff
@@ -1,69 +1,137 @@
+
+The goal of the flight segmentation is to provide a commonly agreed upon, consistent, and easy-to-use subsampling functionality that can be used by reserachers who work with the ORCESTRA measurement data. For instance, all studies that analyse data taken on circular flight paths can use the flight segmentation to select segments of kind `circle`, and are thereby guaranteed to refer to the same commonly agreed upon subset of the campaign data.
+
+All segments belonging to a specific research flight are specified in a yaml-file (stored in `flight_segment_files`) whose name corresponds to the respective flight-ID (e.g. `HALO-20240813a.yaml`). While each segment, defined by a given start and end time, is uniquely labelled with a `segment-ID`, any given time during a flight may belong to one or several segments. For instance, segments of kind `ec_underpass` typically lie within another segment of kind `straight_leg`. This allows flights to be segmented in multiple ways and at multiple levels of granularity. However, as a convention, it is advised that a time or time window may not belong to more than one segment _of the same kind_. Apart from the example of overpasses and underpasses, the first version of the flight segmentation will not contain overlapping segments and coarser segments can be added by collaborators lateron as they see fit.
+
+## Structure of yaml-files
+Every yaml-file begins with a header consisting of the flight's `nickname` (taken [flight reports](https://github.com/orcestra-campaign/book/tree/main/orcestra_book/reports) or adopted freely where necessary), `mission` (ORCESTRA), `platform` (HALO, ATR, METEOR,..), `flight_id` (e.g. HALO-20240813a), and `segments` (which lists all identified flight segments).
+
+The general structure of a segment within the yaml files looks as follows:
+- `kinds`: Can be thought of as tags which label coherent characteristics of the flight segment. See a list of all tags currently in use below.
+- `name`: Short (1 - 3 words) qualitative description of the segment, does not have to be unique
+- `segment_id`: unique identifier of the segment, constructed as the combination of the `flight_id` and the last four digits of a hash computed from the start and end time of the segment
+- `start`: start time of the segment in format `YYYY-MM-DD HH:MM:SS`, e.g. 2024-08-13 14:56:37. 
+- `end`: end time of the segment in same format as start time. 
+- `irregularities`: lists irregularities such as deviations from envisioned flight track due to deep convection, or circles in which no sondes were dropped. Generally meant to be a free text field for proper explanations, this category may contain some standardized *irregularity tags* (**to be decided on**) for automatic checking, and these should be prepended to the explanatory string of the irregularity.
+- `comments`: lists custom comments such as the distance to the exact EC overpass position in the case of an `ec_overpass` segment
```

Comment:
It would be good to add that `segment_id`, `start` and `end` are mandatory, while the rest is optional.
```suggestion
Mandatory components:
- `segment_id`: unique identifier of the segment, constructed as the combination of the `flight_id` and the last four digits of a hash computed from the start and end time of the segment
- `start`: start time of the segment in format `YYYY-MM-DD HH:MM:SS`, e.g. 2024-08-13 14:56:37. 
- `end`: end time of the segment in same format as start time. 
Optional components:
- `kinds`: Can be thought of as tags which label coherent characteristics of the flight segment. See a list of all tags currently in use below.
- `name`: Short (1 - 3 words) qualitative description of the segment, does not have to be unique
- `irregularities`: lists irregularities such as deviations from envisioned flight track due to deep convection, or circles in which no sondes were dropped. Generally meant to be a free text field for proper explanations, this category may contain some standardized *irregularity tags* (**to be decided on**) for automatic checking, and these should be prepended to the explanatory string of the irregularity.
- `comments`: lists custom comments such as the distance to the exact EC overpass position in the case of an `ec_overpass` segment
```